lib/open_food_network/available_payment_method_filter.rb

```diff
@@ -15,9 +15,9 @@ def stripe_enabled?
     end
 
     def stripe_configuration_incomplete?(payment_method)
-      return true if payment_method.preferred_enterprise_id.zero?
-
-      payment_method.stripe_account_id.blank?
+      payment_method.preferred_enterprise_id.nil? ||
+        payment_method.preferred_enterprise_id.zero? ||
+        payment_method.stripe_account_id.blank?
```

Comment:
Isn't this equivalent to `payment_method.preferred_enterprise_id.zero? || payment_method.stripe_account_id.blank?` since

```ruby
irb(main):002:0> nil.blank?
=> true
```

Reply:
no because
2.3.7 :001 > nil.zero?
NoMethodError: undefined method `zero?' for nil:NilClass
	from (irb):1
	from /Users/luisramos/.rvm/rubies/ruby-2.3.7/bin/irb:11:in `<main>'

it's 2 different properties: preferred_enterprise_id and stripe_account_id